front/src/applications/operationalStudies/components/ManageTrainSchedule/AddTrainSchedule.tsx

```diff
@@ -15,16 +15,22 @@ import formatConf from 'applications/operationalStudies/components/ManageTrainSc
 import trainNameWithNum from 'applications/operationalStudies/components/ManageTrainSchedule/AddTrainSchedule/trainNameHelper';
 import { scheduleURL } from 'applications/operationalStudies/components/SimulationResults/simulationResultsConsts';
 import { MANAGE_TRAIN_SCHEDULE_TYPES } from 'applications/operationalStudies/consts';
+import { getConf, getTimetableID } from 'reducers/osrdconf/selectors';
 import getTimetable from '../Scenario/getTimetable';
 
-export default function AddTrainSchedule(props) {
-  const { setDisplayTrainScheduleManagement } = props;
-  const [name, setName] = useState(undefined);
+type Props = {
+  setDisplayTrainScheduleManagement: (type: string) => void;
+};
+
+export default function AddTrainSchedule({ setDisplayTrainScheduleManagement }: Props) {
```

Comment:
```suggestion
export default function AddTrainSchedule({ setDisplayTrainScheduleManagement }: {
  setDisplayTrainScheduleManagement: (type: string) => void;
}) {
```
Can inline the props type (useful since the props type is not exported here)

Reply:
I'am not fine is the type described just above. By the way the type can be exported later. Finally, this pattern (prop Type/interface just before the FC can be repetaed for better redability

Reply:
Hum but we discussed about this yesterday or last week, Alexis told us that he would preferably use inline typing when possible, and we decided to use interfaces if not possible.
As you wish, you can either inline the props or use interface (but not type then) ;)

Reply:
Ok then, sorry I lost the info